css/styles.css

```diff
@@ -400,3 +409,129 @@ button,
 .d-grid {
   display: grid;
 }
+
+@media screen and (min-width: 992px) {
+  /* Body limited to 1100px */
+
+  html {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  body {
+    max-width: 1100px;
+  }
+
+  .body {
+    min-width: 480px;
+  }
+
+  /* Header */
+
+  .navbar {
+    display: none;
+  }
+
+  .navbar-desktop {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .navbar-desktop ul {
+    gap: 40px;
+  }
+
+  .headline {
+    background-image: url(../img/background/headerbg.svg);
+    margin-bottom: 80px;
+    background-repeat: no-repeat;
+  }
+
+  .headline-content {
+    max-width: 700px;
+    margin: 180px 120px 150px 120px;
+    gap: 20px;
+  }
+
+  .headline-content h2 {
+    margin-top: 10px;
+  }
+
+  /* Portfolio - Cards */
+
+  .work-card {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto;
+    gap: 50px;
+  }
+
+  #work-2 .card-info,
+  #work-4 .card-info {
+    grid-column-start: 1;
+    grid-row-start: 1;
+  }
+
+  .my-works img {
+    max-width: 480px;
+  }
+
+  /* About Section */
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1.5fr 2fr;
+    margin: 100px 0 120px 0;
+    gap: 50px;
+  }
+
+  .about-content {
+    margin-left: 50px;
+  }
+
+  #languages-list {
+    flex-direction: row;
+  }
+
+  .language {
+    flex-direction: column;
```

Comment:
- [ ] As I mentioned above it looks different from the original design. I kindly suggest you fix this issue as well.


| Original      | Yours |
  | ----------- | ----------- |
|![Screen Shot 2021-12-01 at 3 14 50 PM](https://user-images.githubusercontent.com/69011963/144218092-7c6eb863-c9ce-44dc-96e6-fc3a2a1cc236.png)|![Screen Shot 2021-12-01 at 3 27 42 PM](https://user-images.githubusercontent.com/69011963/144217966-fec161b1-be60-46e1-b91c-22941c70bc74.png))|

Reply:
@Rustamxon7 Thank you! Will do it ASAP. 😃